app/forms/DateUtils.scala

```diff
@@ -22,11 +22,14 @@ import java.time.format.DateTimeFormatter
 object DateUtils {
 
   private val FORMAT = DateTimeFormatter.ofPattern("d MMM yyyy")
+  private val FORMAT_FULL = DateTimeFormatter.ofPattern("d MMMM yyyy")
```

Comment:
Should we be using consistent date format across all pages? I had spoken to Darryl and he was happy with either MMM or MMMM and we decided to use MMM as it saves space especially on smaller devices

Reply:
Sure, I will remove this. Makes the code cleaner.

Reply:
Done